src/SFA.DAS.AssessorService.Web.Staff/Controllers/Roatp/AddRoatpOrganisationController.cs

```diff
@@ -138,8 +138,8 @@ private Organisation CreateOrganisationFromModel(AddOrganisationViewModel model)
             var organisation = new Organisation
             {
                 Id = Guid.NewGuid(),
-                LegalName = model.LegalName,
-                TradingName = model.TradingName,
+                LegalName = model.LegalName.ToUpper(),
+                TradingName = model.TradingName?.ToUpper(),
```

Comment:
Minor quibble but I'll leave this to you.  We store the legal name in upper case, but Trading name is usually stored in pascal case. Not sure where that convention came from, but it seems to be the pattern?  I'll leave it with you

Reply:
It's a requirement in the user story to store both legal name and trading name in upper case